index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Chris Duke!!!</title>
+
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.6.3/css/font-awesome.css">
+  <link rel="stylesheet" href='https://fonts.googleapis.com/css?family=Raleway'
+</head>
+<body>
+  <nav class="top-nav">
+    <ul class="nav-list">
+      <li class="nav-list__item nav-list__item--brand">My Portfolio</li>
+      <li class="nav-list__item">About</li>
+      <li class="nav-list__item">Day One</li>
+      <li class="nav-list__item">Cool Stuff</li>
+    </ul>
+  </nav>
+  <header class="hero">
+    <div class="section-container hero__container">
+      <h2 class="hero__title">This is my portfolio!</h2>
+      <hr class="divider divider--hero">
+      <p class="hero__text">Please enjoy.</p>
+    </div>
+  </header>
+  <section class="bg-pop">
+    <div class="nav-list__item--brand">
+      <h3 class="section-header">I'm a student at the Iron Yard</h3>
+      <hr class="divider">
+      <p>I'm not sure what I'm doing.</p>
+    </div>
+  </section>
+  <section class="bg-standard">
+    <div class="nav-list__item--brand">
+      <h3 class="section-header">My First Day</h3>
+      <hr class="divider">
+      <p>Day one was fun and challenging.
+        I found the homework assignment to be difficult.
+        I was unable to remember much of the CSS I learned in the pre-work. </p>
+    </div>
+  </section>
+  <section id="bg-standard">
+    <div class="section-container bg-photo">
```

Comment:
The `bg-something` classes will only be on the `section` elements and the container will be within it.